akka-cluster-tools/src/test/scala/akka/cluster/pubsub/DistributedPubSubMediatorRouterSpec.scala

```diff
@@ -16,6 +16,7 @@ object DistributedPubSubMediatorRouterSpec {
   def config(routingLogic: String) = s"""
     akka.loglevel = INFO
     akka.actor.provider = "cluster"
+    akka.actor.serialize-messages = off
```

Comment:
Didn't figure this one out, but as there is multi-jvm tests it should be ok to not verify that _all_ the internal messages can be sent?